.github/workflows/nlu-hyperopt.yml

```diff
@@ -0,0 +1,17 @@
+
+name: main
+on: [pull_request]
+
+jobs:
+  nlu-hyperopt:
+    name: NLU hyperoptimization
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: RasaHQ/nlu-hyperopt@rasa-1-compat
+      name: Run NLU Hyperoptimization
+      env:
+        DATA_DIR: $GITHUB_WORKSPACE/data
+        SEARCH_SPACE: $GITHUB_WORKSPACE/nlu_hyperopt/space.py
```

Comment:
This works in our case, but not for users using the action.
What you want to use in your `Dockerfile` are `ARG`s: https://docs.docker.com/engine/reference/builder/#arg
And then use these `ARG`s in the `COPY` commands. 

I believe line 14 (`RUN if ...`) only works because you do `COPY nlu_hyperopt`. However, if the space description would be in a directory `foo`. It wouldn't work.

Reply:
I tested this in another repo https://github.com/melindaloubser1/rasatest/runs/614104189?check_suite_focus=true and as you say, it doesn't work there. Was trying to figure out why.  I went back on forth on the ARGs - couldn't quite figure out how to get them to work with env vars (since you can't set `--build-args` at runtime in the action`). I'll give it another shot.

Reply:
Question: Why would `MAX_EVALS` register correctly in the container, but not `DATA_DIR` or `SEARCH_SPACE` ? from the test PR above I can see that it is not picking up the env var at all

Reply:
Ok, it works on the test repo too now. But, I had to change the names of all the env variables to match what Github will produce i.e. `INPUT_<var>`. And it now does  an import directly from the absolute path of the file at `INPUT_SEARCH_SPACE` path. Is that ok? It's a hack I've used before but I have a feeling it's not really best practice.

Reply:
just saw the `build args` are not passed to the Docker container. Did you also read this section in the github actions docs https://help.github.com/en/actions/building-actions/dockerfile-support-for-github-actions#workdir ? Apparently it mounts the `GITHUB_WORKING_DIRECTORY` into the docker container.

Reply:
What do you mean by "hack"? The fact that you're loading a python module during runtime?

Reply:
Ja I was told (possibly wrongly) that it's a hack to load a module from an absolute file path. But if it's not then great.

Reply:
Yes, I saw about the `GITHUB_WORKING_DIRECTORY` - figured people mostly still need to set the path relative to that where their files are (i.e. it won't be in `./data`). By setting the inputs it will find the files from the env vars